From 7661d31e3e4595276a6844f3b92ad34204b4a894 Mon Sep 17 00:00:00 2001
From: Judith Mendez <jm@ti.com>
Date: Thu, 22 Feb 2024 16:03:25 -0600
Subject: [PATCH 06/27] arm64: dts: ti: k3-am64-main: Fix OTAPDLY value for HS
 timing

The sdhci1 node for am64-main has the wrong OTAPDLY value
for hs timing, fix this according to the datasheet [0].

[0] https://www.ti.com/lit/ds/symlink/am6422.pdf

Signed-off-by: Judith Mendez <jm@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am64-main.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am64-main.dtsi b/arch/arm64/boot/dts/ti/k3-am64-main.dtsi
index dbc0a9cc67a5..fdc6f59346e6 100644
--- a/arch/arm64/boot/dts/ti/k3-am64-main.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am64-main.dtsi
@@ -498,7 +498,7 @@ sdhci1: mmc@fa00000 {
 		ti,itap-del-sel-sdr12 = <0x0>;
 		ti,itap-del-sel-sdr25 = <0x0>;
 		ti,otap-del-sel-legacy = <0x0>;
-		ti,otap-del-sel-sd-hs = <0xf>;
+		ti,otap-del-sel-sd-hs = <0x0>;
 		ti,otap-del-sel-sdr12 = <0xf>;
 		ti,otap-del-sel-sdr25 = <0xf>;
 		ti,otap-del-sel-sdr50 = <0xc>;
-- 
2.39.2

